index.js

```diff
@@ -116,6 +116,10 @@ function createPreprocessor(config, basePath, logger) {
         log.debug('Replacing require "%s" with "%s"', moduleName, modulePath);
```

Comment:
note that this is a workaround to a more fundamental issue with the code.

When you are writing new javascript content, you need to properly escape strings:

```javascript
'window.__cjsModuleAliases["' + moduleName + '"] = "' + modulePath + '";';
```

...Should use JSON.stringify:

```javascript
'window.__cjsModuleAliases[' + JSON.stringify(moduleName) + '] = ' + JSON.stringify(modulePath) + ';';
```

... but i've left those issues to be addressed separately because it seems there are many places in this code that assume forward slashes, so the least risky fix here is to simply normalize.